avaframe/in3Utils/fileHandlerUtils.py

```diff
@@ -0,0 +1,143 @@
+"""
+    Directory and file handling helper functions
+
+    This file is part of Avaframe.
+"""
+
+# Load modules
+import os
+import glob
+import logging
+import numpy as np
+import shutil
+
+# create local logger
+# change log level in calling module to DEBUG to see log messages
+log = logging.getLogger(__name__)
+
+
+def makeADir(dirName, flagKeepDir=False):
+    """ Make directories """
+
+    # If directory already exists - Delete directory first is default
+    if os.path.isdir(dirName):
+        if flagKeepDir:
```

Comment:
so you keep it this way where the default is remove dir?

Reply:
What is your opinion, shall I put it the other way round and make a FlagRemDir?

Reply:
okay lets keep it like this

Reply:
I think you are right, I changed it to flagRemDir